public/index.html

```diff
@@ -2,42 +2,10 @@
 <html lang="en">
```

Comment:
웹 접근성 향상을 위해 웹페이지의 콘텐츠가 한국에서 사용되는 한국어로 작성되었다는 것을 알려주세요! 
```suggestion
<html lang="ko">
```